src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter.Time.cs

```diff
@@ -19,15 +19,16 @@ public static bool TryFormat(this DateTimeOffset value, Span<byte> buffer, out i
                 offset = value.Offset;
             }
 
-            Precondition.Require(format.Symbol == 'R' || format.Symbol == 'O' || format.Symbol == 'G');
-
             symbolTable = symbolTable ?? SymbolTable.InvariantUtf8;
 
             switch (format.Symbol)
             {
                 case 'R':
                     return TryFormatDateTimeRfc1123(value.UtcDateTime, buffer, out bytesWritten, symbolTable);
 
+                case 'r':
```

Comment:
The whole point of this addition/PR is that for Azure authentication token I need to have a lowercased datetime. i.e. this PR adds 'r' to mean: "just like R but lowercase"